elastic/kibana
# [Monitoring] Fix Set Up Monitoring redirect to setup mode

## Summary

Fixes #112215 

To test:

1. Set `monitoring.ui.render_react_app: true` in `kibana.dev.yml`.
2. Start Kibana and connect it to a **local** cluster running from ES snapshot. **Do not start Metricbeat or ingest any data**
3. Open the Monitoring app and ensure the No Data screen displays as expected
4. Click **Set Up Monitoring With Metricbeat** and ensure you are redirected to the Nodes page in Setup Mode
5. Refresh the Nodes page in Setup Mode. Ensure that `inSetupMode:!t` remains in the URL, and that the Nodes page successfully refreshes without crashing.
6. Turn off Setup Mode and ensure you are redirected back to the No Data page
